pyproject.toml

```diff
@@ -1,7 +1,55 @@
+[tool.poetry]
+name = "python-pachyderm"
+version = "7.2.0"
+description = "Python Pachyderm Client"
+authors = ["Pachyderm <python-pachyderm@pachyderm.io>"]
```

Comment:
Let's use the email `integrations@pachyderm.io` instead. @albscui thoughts?

Reply:
I grabbed this from [this issue](https://github.com/pachyderm/python-pachyderm/issues/280) on a whim. Happy to change it to the correct one

Reply:
Great! This integrations email is the one we're using for managing all integrations-related projects moving forward.